src/zeek.bif

```diff
@@ -16,6 +16,8 @@
 #include <cstdio>
 #include <time.h>
 
+#include <ares_nameser.h>
```

Comment:
I can't say I have a really good argument against defining them ourselves other than that those values are standardized and the ares header has the entire list already available to us.

Reply:
Ok, really just goes back to whether we will have the c-ares headers available with an installation tree.

Reply:
I went ahead and removed the dependency and recreated the type and macro defines in DNS_Mgr.h. I can't think of when you'd need to install the header since DNS_Mgr is all internally-namespaced and so you shouldn't be including DNS_Mgr.h directly, but I could be wrong. It's easier to just not have it there.